velox/functions/prestosql/Bitwise.h

```diff
@@ -175,6 +175,15 @@ struct BitwiseLogicalShiftRightFunction {
     // Presto defines this only for bigint, thus we will define this only for
     // int64_t.
     if (bits == 64) {
+      if (number < 0) {
```

Comment:
Do we need to do something similar for left shift ?

Reply:
For left shift, logical and arithmetic behave the same.